From 9e50c0d23a92c70cece88c47dc4e23c09e1d1ea9 Mon Sep 17 00:00:00 2001
From: Viorel Cernateanu <vrilutza@gmail.com>
Date: Sat, 24 Oct 2026 09:46:15 +0300
Subject: [PATCH] avb: fix argument type for Long in choice range

Use 0LL instead of 0 for the minimum value in SPA_POD_CHOICE_RANGE_Long
to prevent SIGSEGV errors on 32-bit architectures.

This is the same fix as commit 5a023c8c8 for issue #2271 (alsa).

diff --git a/spa/plugins/avb/avb-pcm.c b/spa/plugins/avb/avb-pcm.c
index 22d7770da..06d917d9a 100644
--- a/spa/plugins/avb/avb-pcm.c
+++ b/spa/plugins/avb/avb-pcm.c
@@ -256,7 +256,7 @@ struct spa_pod *spa_avb_enum_propinfo(struct state *state,
 			SPA_PROP_INFO_name, SPA_POD_String("latency.internal.ns"),
 			SPA_PROP_INFO_description, SPA_POD_String("Internal latency in nanoseconds"),
 			SPA_PROP_INFO_type, SPA_POD_CHOICE_RANGE_Long(state->process_latency.ns,
-				0, 2 * SPA_NSEC_PER_SEC),
+				0LL, 2 * SPA_NSEC_PER_SEC),
 			SPA_PROP_INFO_params, SPA_POD_Bool(true));
 		break;
 	case 15:

